### PR TITLE
LPS-124337 update only the changed field property in state

### PR DESCRIPTION
**Pull request by marcosapmf**
Related issue: https://issues.liferay.com/browse/LPS-124337

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
cc/ @interaminense, @kevenleone, @oliveiraaraujo, @marcelabc, @jeyvison

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/data/engine/content/type/DLDataDefinitionContentType.java [hunk 1]
```diff
@@ -41,6 +41,11 @@
 )
 public class DLDataDefinitionContentType implements DataDefinitionContentType {
 
+	@Override
+	public boolean allowEmptyDataDefinition() {
```
Comment (jeyvison) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/data/engine/content/type/DLDataDefinitionContentType.java:45]: The commit message is different from what you're trying to do here. Whats the right one?

**Review by jeyvison — COMMENTED** [review]
Hey Marcos. Please see my comment. In general , LGTM.

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/data/engine/content/type/DLDataDefinitionContentType.java (hunk 1 above)
Comment (marcosapmf) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/data/engine/content/type/DLDataDefinitionContentType.java]: @jeyvison my bad. I implemented the wrong method here :sweat_smile:

**Comment by oliveiraaraujo** [comment]
LGTM!

**Comment by jeyvison** [comment]
LGTM!

**Comment by marcosapmf** [comment]
ci:test:sf

**Review by jeyvison — APPROVED** [review]
ci:forward

**Comment by marcosapmf** [comment]
ci:forward

**Comment by marcosapmf** [comment]
ci:forward

**Comment by marcosapmf** [comment]
ci:forward

**Comment by marcosapmf** [comment]
ci:forward

**Comment by marcosapmf** [comment]
ci:stop

**Comment by marcosapmf** [comment]
ci:forward

**Comment by marcosapmf** [comment]
ci:forward

**Comment by marcosapmf** [comment]
ci:forward

**Comment by marcosapmf** [comment]
ci:forward

**Comment by marcosapmf** [comment]
ci:forward

**Comment by marcosapmf** [comment]
ci:forward

**Comment by marcosapmf** [comment]
ci:forward

**Comment by marcosapmf** [comment]
resend https://github.com/liferay-data-engine/liferay-portal/pull/52

**Pull request closed without merging**